Python/generated_cases.c.h

```diff
@@ -63,6 +63,145 @@
             DISPATCH();
         }
 
+        TARGET(LOAD_FAST__LOAD_FAST) {
+            PyObject *_tmp_1;
+            PyObject *_tmp_2;
+            {
```

Comment:
Random idea... any chance we could annotate the output of macros to make it easier to see what the different parts correspond to? So, like, this line would be:

```suggestion
            {  // LOAD_FAST
```

And we would have `{  // JOIN` and another `{  // LOAD_FAST` below? Not sure how hard this is.